content/post/articles/Music-and-silence.md

```diff
@@ -0,0 +1,57 @@
+
+###### Boost creativity
+
+Creativity can be boosted by music, according to a [study](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0182210) by researchers at Radboud University (Netherlands) and Sydney University of Technology (Australia) the researchers found that listening to happy music, which they define as music with positive, awakening values (a relative positivity because coders who love devil metal are also happy) facilitates more inspiring creative thinking than silence.
+
+I'm a supporter of this opinion, so here are some nuggets(without wanting to advertise) I use when I meet mister bug:
+- [RISE (ft. The Glitch Mob, Mako, and The Word Alive)](https://www.youtube.com/watch?v=fB8TyLTD7EE)
+- [Phoenix (avec Cailin Russo et Chrissy Costanza)](https://www.youtube.com/watch?v=i1IKnWDecwA)
+- [Take Over (avec Jeremy McKinnon (A Day To Remember), MAX, Henry)](https://www.youtube.com/watch?v=KbNL9ZyB49c)
+
+# Code with silence
+
+Lao Tzu said <<*Silence is a source of Great Strength*>>
+
+Not everyone uses music to cope with the different difficulties that one can encounter while coding so there are those who are for silence (the zone), a 2013 [study](https://www.researchgate.net/publication/259110014_Is_silence_golden_Effects_of_auditory_stimuli_and_their_absence_on_adult_hippocampal_neurogenesis) found that two hours of silence could create new cells in the hippocampus region, a brain area linked to learning, remembering, and emotions.According to a 2006 [study](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1860846/), two minutes of silence relieves tension in the body and brain and is more relaxing than listening to music.  
+Silence also has its advantages:
+
+###### Boost creativity
+When you allow thoughts to go where they want, inspiration can flow. Solutions to current or long standing problems may suddenly appear to you, or a workaround or innovative approach may seem more achievable. Ideas for going in a different direction might merge together, help build momentum and enthusiasm to turn them into other potential avenues to pursue.Follow this [link](https://psychcentral.com/blog/the-hidden-benefits-of-silence#1) to read the article.  
+
+###### Deal with your emotions
+When you practice solitude and silence, you allow your brain to process your emotions, rather than suppressing them and staying on autopilot. You may experience uncomfortable emotions during this process, but the sooner you recognize and process your feelings, the sooner you can move forward in health and freedom.  
+Follow this [link](https://www.piedmont.org/living-better/the-benefits-of-silence-and-solitude) to read the article.
+
+
+In the end, I think that to improve creativity, concentration when coding, it really depends on everyone's desires and affinities, whether with music or with silence (the zone)🔥.  
+
+Thanks for reading. Here are my sources:
```

Comment:
Mentioning articles you already linked in the body of the article is a bit redundant, don't you think?